### ISSUE TITLE
1.x: CRI-O CI is broken due to https://github.com/cri-o/cri-o/issues/4770

### ISSUE THREAD

**Issue by fidencio** [kata-containers/tests#3423]
Since https://github.com/cri-o/cri-o/issues/4506 got merged, CRI-O default config file is empty, unless use explicitly generate it with `crio config --default`.

For 2.x, I have a PR working on a better way, for us, to change the default config instead of using `sed` in the default values.  However, as the 1.x will reach its EOL in a month or so, I don't think it's worth it doing something else than the bare minimum to get it working.

**Comment by fidencio** [comment]
This one is not yet fixed by #3424.  I'll keep debugging it.

**Comment by fidencio** [comment]
Okay, the other issue faced was related to the critools version being used in the tests and I solved that by changing the jenkins tests directly, and there I added:
```
critools_version="${branch_release_number}.0"
if [ "$branch_release_number" == "master" ]; then
	critools_version="1.21.0"
fi
```

That will ensure that CRI-O master will be tested using critools_version=1.21.0, and other branches with the critools aligned with the CRI-O version.

For now I'm closing this one.
